liburing/build.rs

```diff
@@ -0,0 +1,45 @@
+extern crate bindgen;
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+
+fn main() {
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    Command::new("cp")
+        .arg("-r")
+        .arg("lib")
+        .arg(out_dir.clone())
+        .status()
+        .expect("copy liburing to out_dir");
+    Command::new("make")
+        .arg("liburing.a")
```

Comment:
I think we can't statically link liburing because of LGPL, maybe you can consider using [linux-io-uring](https://github.com/quininer/linux-io-uring).